src/core/tests/type_prop/paged_attention.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/paged_attention.hpp"
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/test_assertions.hpp"
+#include "common_test_utils/type_prop.hpp"
+#include "openvino/openvino.hpp"
+#include "openvino/opsets/opset13.hpp"
```

Comment:
Remove it include required operators.

Reply:
Done

Reply:
Still not changed but can be optionally done later.

Reply:
Removed the opset13.hpp and replaced it with the direct header for `Parameter`